app/utils/whitelabel.ts

```diff
@@ -1,31 +1,89 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
+import icon from '../assets';
+import styles from '../components/ui/Navigation/Navigation.module.scss';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
+// Include new white label here
+type WhiteLabelSiteKey = 'default' | 'SFServiceGuide' | 'SFFamilies';
 
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
+interface WhiteLabelSite {
+  title: string;
+  siteUrl: string;
+  appImages: {
+    background: string;
+     // For appImages, logoLarge and logoSmall are usually different between each white label
```

Comment:
I'm actually a little bit confused by this comment, since all of the whitelabel configurations use the same image for both `logoLarge` and `logoSmall`. What did you mean by "usually different"?